src/Stan/Report/Html.hs

```diff
@@ -41,334 +40,318 @@ import Stan.Severity (Severity (..), severityDescription)
-              )
-              )
-        # li_A (A.class_ "col-12 obs-li") (divClass "observations col-12"
-            ( h4_ "Observations" # map stanObservation (toList fileInfoObservations)))
-        )
-    )
-
-stanExtensions from exts = divClass "col-6"
-    ( button_A (A.class_ "collapsible") ("Extensions from " # from)
-    # ol_A (A.class_ "content") (map li_ exts)
-    )
-
-inspectionLink ins = a_A (A.class_ "ins-link" # A.href_ (toText "#" <> insId)) insId
+        Nothing -> liSum $ do
+            h4 "Congratulations! Your project has zero vulnerabilities!"
+            span "Stan carefully run all configured inspection and found 0 observations and vulnerabilities to the project"
+        Just Summary{..} -> do
+            liSum $ do
+                h4 $ toHtml ("Watch out for " <> unId summaryInspectionId)
+                span $ do
+                    toHtml @Text "By the result of Stan analysis, the most common inspection for this project is "
+                    inspectionLink summaryInspectionId
+            liSum $ do
+                h4 $ toHtml ("Vulnerable module: " <> unModuleName summaryModule)
+                span $ do
+                    toHtml @Text "The "
+                    code (toHtml $ unModuleName summaryModule)
+                    toHtml @Text " module is the most vulnerable one in the project, as it got the most number of observations"
+            liSum $ do
+                h4 (toHtml $ "Popular category: " <> unCategory summaryCategory)
+                categories "inline" $ one summaryCategory
+                toHtml @Text "The project has the most problems with inspections from this category"
+            liSum $ do
+                h4 $ toHtml ("Severity: " <> show @Text summarySeverity)
+                toHtml @Text "The highest severity of found vulnerabilities is "
+                severity (show @Text summarySeverity)
+
+    liSum :: Html -> Html
+    liSum = li ! (A.class_ "sum")
+
+stanObservations :: Analysis -> Html
+stanObservations Analysis{..} = do
+    divClass "row" (blockP "List of found vulnerabilities per file")
+    sequence_ $ map stanPerFile $
+        filter (not . null . fileInfoObservations) $ Map.elems analysisFileMap
+
+stanPerFile :: FileInfo -> Html
+stanPerFile FileInfo{..} = divIdClass "file" "row" $ do
+    h3 ! (A.class_ "grey-bg") $ toHtml $ "📄 " <> fileInfoPath
+    ul $ do
+        li $ tableWithShadow "col-6" $ do
+            tableRow "Module" $ code $ toHtml $ unModuleName fileInfoModuleName
+            tableRow "Lines of Code" fileInfoLoc
+        li $ divClass "extensions" $ do
+            stanExtensions ".cabal" (extensionsToText fileInfoCabalExtensions)
+            stanExtensions "module" (extensionsToText fileInfoExtensions)
+        li ! (A.class_ "col-12 obs-li") $ divClass "observations col-12" $ do
+            h4 "Observations"
+            sequence_ $ map stanObservation $ toList fileInfoObservations
```

Comment:
Also should look much simpler now, since `Slist` has `Foldable` instance :slightly_smiling_face: 
```suggestion
            traverse_ stanObservation fileInfoObservations
```